js/job.js

```diff
@@ -0,0 +1,47 @@
+var jobs = [];
+var projects = [];
+
+function Job (opt) {
+  this.company = opt.company;
+  this.position = opt.position;
+  this.dates = opt.dates;
+  this.responsibilities = opt.responsibilities;
+}
+
+Job.prototype.toHtml = function() {
+  var source = $('#job-template').html();
+  var template = Handlebars.compile(source);
+  return template(this);
+};
+$.getJSON('data/jobHistory.json', function(data){
```

Comment:
What's this random AJAX call doing in the middle of the script? It should be in a method you can call.